packages/itwin/ec3-widget/package.json

```diff
@@ -0,0 +1,114 @@
+{
+  "name": "@itwin/ec3-react",
+  "version": "1.0.0",
```

Comment:
Downgrade version to 0.1.0 for initial (pre)release.

Reply:
I believe it should just be 0.0.0 and let the pipeline bump it for us with `rush change`

Reply:
Changed the version to 0.0.0